src/table/matrix_table.cpp

```diff
@@ -90,6 +90,19 @@ void MatrixWorkerTable<T>::Get(const std::vector<integer_t>& row_ids,
 }
 
 template <typename T>
+void MatrixWorkerTable<T>::Get(T* data, size_t size, integer_t* row_ids,
```

Comment:
You may take a look at other Get function I implemented before. The size means differently